contracts/EuroToken.sol

```diff
@@ -0,0 +1,235 @@
+pragma solidity 0.4.15;
+
+import './AccessControl/AccessControlled.sol';
+import './Math.sol';
+import './Reclaimable.sol';
+import './Standards/IERC677Token.sol';
+import './Standards/IERC677Callback.sol';
+import './Standards/ITokenWithDeposit.sol';
+import './SnapshotToken/Helpers/TokenMetadata.sol';
+import './Zeppelin/StandardToken.sol';
+import './MigrationSource.sol';
+import './EuroTokenMigrationTarget.sol';
+
+
+/// Simple implementation of EuroToken which is pegged 1:1 to certain off-chain
+/// pool of Euro. Balances of this token are intended to be migrated to final
+/// implementation that will be available later
+contract EuroToken is
+    ITokenWithDeposit,
+    IERC677Token,
+    AccessControlled,
+    StandardToken,
+    TokenMetadata,
+    MigrationSource,
+    Reclaimable
+{
+    ////////////////////////
+    // Constants
+    ////////////////////////
+
+    string private constant NAME = "Euro Token";
+
+    string private constant SYMBOL = "EUR-T";
+
+    uint8 private constant DECIMALS = 18;
+
+    ////////////////////////
+    // Mutable state
+    ////////////////////////
+
+    // a list of addresses that are allowed to receive EUR-T
+    mapping(address => bool) private _allowedTransferTo;
+
+    // a list of of addresses that are allowed to send EUR-T
+    mapping(address => bool) private _allowedTransferFrom;
```

Comment:
If we want to use this contract longer, I would turn it into something more robust, like calling an external contract.

Reply:
This contract should be replaced when we have KYC and bank details verification. `_allowedTransferTo` will be replaced with a contract that holds KYC results or some other client registry. We agreed with Andre and Jacek for extremely policed version of EUR-T at start